Amethyst/Layout/Layouts/CustomLayout.swift

```diff
@@ -49,6 +49,12 @@ class CustomLayout<Window: WindowType>: StatefulLayout<Window> {
             return nil
         }
 
+        context.evaluateScript("var console = { log: function(message) { _consoleLog(message) } }")
+        let consoleLog: @convention(block) (String) -> Void = { message in
+            NSLog(message)
```

Comment:
Can you use the logging used elsewhere rather than directly to syslog?

Reply:
```    func applicationDidFinishLaunching(_ notification: Notification) {
        #if DEBUG
            log.addDestination(ConsoleDestination())
        #endif

        log.info("Logging is enabled")```
This tells me that logging is only enabled for debug builds, right? I don't think using console.log to debug a custom layout should require a debug build of Amethyst?

Reply:
There used to be a command-line option for sending logs to stdout. Looks like I lost it somewhere along the way.

Reply:
@ianyh added `--log` command line option and switched to the logging framework used by you.